build-scripts/components/runc/build.sh

```diff
@@ -3,5 +3,6 @@
 export INSTALL="${1}/bin"
 mkdir -p "${INSTALL}"
 
+export SHIM_CGO_ENABLED=1
```

Comment:
I'm not sure if this is used by runc. The runc makefile seems to use `CGO_ENABLED` as seen [here](https://github.com/opencontainers/runc/blob/69225fa9199431bf3656766c8a1274473e4bb923/Makefile#L49-L56). Maybe we can include both lines just to be safe.